mnefun/_report.py

```diff
@@ -251,6 +252,37 @@ def _report_raw_psd(report, raw, raw_pca=None, p=None):
     print('%5.1f sec' % ((time.time() - t0),))
 
 
+def _peak_times(evoked, max_peaks=5):
+    '''Return times of prominent peaks from global field power.'''
+    gfp_list = []
+    if 'meg' in evoked:
+        evk = evoked.copy().pick_types(meg=True)
+        gfp = evk.data.std(axis=0)
+        gfp_list.append(gfp/gfp.max())
```

Comment:
by local convention we follow the PEP8 spacing between separators
```suggestion
        gfp_list.append(gfp / gfp.max())
```

Reply:
... and this is why Travis is unhappy:

https://travis-ci.org/github/LABSN/mnefun/builds/665584883#L393-L406